ouroboros-network-framework/src/Ouroboros/Network/ConnectionId.hs

```diff
@@ -0,0 +1,21 @@
+{-# LANGUAGE DerivingVia #-}
```

Comment:
This file extracted from a different file, so that `ConnectionId` is more widely available.